types/src/model/util.rs

```diff
@@ -4,3 +4,35 @@
 //!
 //! These structs model the types returned by the JSON-RPC API but have concrete types
 //! and are not specific to a specific version of Bitcoin Core.
+
+use serde::{Deserialize, Serialize};
+
+/// Models the result of JSON-RPC method `createmultisig`.
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct CreateMultisig {
+    /// The value of the new multisig address.
+    pub address: String,
+    /// The string value of the hex-encoded redemption script.
+    #[serde(rename = "redeemScript")]
+    pub redeem_script: String,
+}
+
+/// Models the result of JSON-RPC method `validateaddress`.
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct ValidateAddress {
+    /// If the address is valid or not. If not, this is the only property returned.
+    pub isvalid: bool,
+    /// The bitcoin address validated.
+    pub address: Option<String>,
+    /// The hex encoded scriptPubKey generated by the address.
+    #[serde(rename = "scriptPubKey")]
+    pub script_pub_key: Option<String>,
+    /// If the key is a script.
+    pub isscript: Option<bool>,
+    /// If the address is a witness address.
+    pub iswitness: Option<bool>,
+    /// The version number of the witness program.
+    pub witness_version: Option<u8>,
+    /// The hex value of the witness program.
+    pub witness_program: Option<String>,
+}
```

Comment:
Same for all other fields, should be types from `rust-bitcoin` and use Rust idiomatic field names.